### PR TITLE
Issue 883 g.calibrate review

### PR DESCRIPTION
**Pull request by l-k-**
Contributes to #883

@vincentvanhees I was working through g.calibrate, and it was a bit hard to follow because of all the differences in the input file formats. So I simplified both g.calibrate and g.getmeta a bit by hiding all the input-format-specific logic within g.readaccfile. This is the main commit of the PR: https://github.com/wadpac/GGIR/commit/75262beec5edde7796442d5bb6843456d7352f84

There are a lot of changes, and I need to do a lot more testing, but I thought I'd open a draft PR to see if you think this is worth doing. On the one hand it's risky to make so many changes, but on the other hand I think the code will be easier to maintain. 

The new test test_greadaccfile.R is currently failing, but only because I haven't pulled in this commit from main: https://github.com/wadpac/GGIR/commit/f1da4569cae4a76d33283164b8404f2dd8646e35 (if I pull it in, this PR will be harder to merge)

Checklist before merging:

- [x] Existing tests still work (check by running the test suite, e.g. from RStudio).
- [x] Added tests (if you added functionality) or fixed existing test (if you fixed a bug).
- [ ] Updated or expanded the documentation.
- [ ] Updated release notes in `inst/NEWS.Rd` with a user-readable summary. Please, include references to relevant issues or PR discussions.
- [ ] Added your name to the contributors lists in the `DESCRIPTION` and `CITATION.cff` files.

### CODE REVIEW

**Comment by vincentvanhees** [comment]
Thanks @l-k- I think the effort makes sense, but I agree that it could be risky.

So, possible approach could be:
- Try to avoid making this work grow into ever more updates, and if possible move some of the updates to a separate pull request to help create focus for this PR.
- Coordinate with Jairo and myself to help test components of this in order to identify problems early on.

I am not sure I understand the following. The commit you refer to is very small. Are you sure the link is to the correct commit? 

> I haven't pulled in this commit from main: https://github.com/wadpac/GGIR/commit/f1da4569cae4a76d33283164b8404f2dd8646e35 (if I pull it in, this PR will be harder to merge)

**Comment by l-k-** [comment]
closing for now, will redo.
